packages/quick-open/src/browser/quick-open.view.tsx

```
@@ -303,64 +331,75 @@ const QuickOpenItemView: React.FC<IQuickOpenItemProps> = observer(({ data, index
-      {(mouseOver || widget.selectIndex === index) &&
+      {(mouseOver || selectIndex === index) &&
         buttons?.map((button) => (
           <QuickOpenButton onMouseDown={(event) => onSelectButton(event, button)} button={button} />
         ))}
     </div>
   );
-});
+};
 
 export const QuickOpenList: React.FC<{
   onReady: (api: IRecycleListHandler) => void;
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)